api/src/api/userRouter.test.ts

```diff
@@ -380,139 +380,149 @@ describe("userRouter", () => {
         mockJwt.decode.mockReturnValue(cognitoPayload({ id: "123" }));
       });
 
-      it("does not update business name search if businessNameAvailability is undefined", async () => {
-        const userData = generateUserDataForBusiness(
-          generateBusiness({
-            formationData: generateFormationData({ businessNameAvailability: undefined }),
-          })
-        );
-        stubUserDataClient.get.mockResolvedValue(userData);
+      describe("when businessPersona is 'STARTING'", () => {
```

Comment:
I think this is fine as long as we still have all of these same cases covered for Dakota. Pulled the branch locally to confirm, and I think that's the case, but if you confirm I'd appreciate it.